cmd/stackdriver-prometheus-sidecar/main.go

```diff
@@ -309,9 +313,9 @@ func main() {
 		}
 	}
 
-	filters, err := parseFilters(cfg.filters...)
+	filtersets, err := parseFiltersets(logger, cfg.filtersets, cfg.filters)
 	if err != nil {
-		level.Error(logger).Log("msg", "Error parsing filters", "err", err)
+		level.Error(logger).Log("msg", "Error parsing filtersets", "err", err)
```

Comment:
Error parsing filter/include - as it is error for user.

Reply:
Thanks, good catch. Fixed!